src/coreclr/src/vm/arm64/asmhelpers.S

```diff
@@ -375,7 +375,7 @@ WRITE_BARRIER_ENTRY JIT_WriteBarrier
     // Update GC Shadow Heap
 
     // Do not perform the work if g_GCShadow is 0
-    ldr  x12, LOCAL_LABEL(wbs_GCShadow)
```

Comment:
is `wbs_GCShadow` still required? Also is this similar to how its done for x64?

Reply:
Nope, we don't need the label anymore, I can remove it.

Reply:
Actually, I think we will leave it and add a comment on it instead.